packages/lesswrong/themes/globalStyles/externalStyles.ts

```diff
@@ -0,0 +1,7 @@
+
+export const globalExternalStylesheets = [
+  "https://fonts.googleapis.com/icon?family=Material+Icons",
+  "https://cdn.jsdelivr.net/npm/instantsearch.css@7.0.0/themes/reset-min.css",
+  "https://fonts.googleapis.com/css?family=Roboto:300,400,500",
```

Comment:
I think you probably know this, but just in case: I think we can straight up drop some of these, which is presumably even better than preloading them. E.g. we only use Roboto in a few places, and we could probably just switch to a different font.